tests/PKSim.Tests/IntegrationTests/IndividualToIndividualBuildingBlockMapperSpecs.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Extensions;
+using PKSim.Core.Mappers;
+using PKSim.Core.Model;
+using PKSim.Infrastructure;
+
+namespace PKSim.IntegrationTests
+{
+   public abstract class concern_for_IndividualToIndividualBuildingBlockMapper : ContextForIntegration<IIndividualToIndividualBuildingBlockMapper>
+   {
+      protected Individual _individual;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         _individual = DomainFactoryForSpecs.CreateStandardIndividual();
```

Comment:
@rwmcintosh the unit test only was just too hard to code. I decided to create an integration test for this instead. So much easier.